gielladetect/text_cat.py

```diff
@@ -36,8 +36,8 @@
 http://odur.let.rug.nl/~vannoord/TextCat/
 """
 
-
-from typing import Optional, Self, TextIO, Tuple
+from typing_extensions import Optional, Self, TextIO, Tuple
+#from typing import Optional, Self, TextIO, Tuple
```

Comment:
pylint and mypy complain about this, try instead

```suggestion
from typing import Optional, TextIO, Tuple
from typing_extensions import Self
```

as well as moving these 2 lines at the end of the import block, and changing python version in the github workflow file.

Reply:
Fixed in new commit